cypress/integration/1-getting-started/todo.spec.js

```diff
@@ -6,23 +6,34 @@ describe('App', () => {
   })
 
   it('should be able to visit the page and render movies', () => {
+    cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies')
     cy.get('.card__poster').should('have.length', 40)
```

Comment:
I would add another `should` here that spot checks that the image is correct on one of the posters